Resources/Prototypes/Objectives/stealTargetGroups.yml

```diff
@@ -347,8 +347,8 @@
     state: ian
 
 - type: stealTargetGroup
-  id: AnimalBingus
-  name: Bingus
+  id: AnimalNamedCat
+  name: Cat
```

Comment:
should be made clear to players that its a special cat not just random 1k cargo order

Reply:
Any suggestions?

Reply:
Named Cat :trollface:

Reply:
Is that clear enough? Seems...just as vague

Reply:
Unique cat?

Reply:
I went with CMO's Cat since that's where the spawner is usually located/and it's in the mapping guide there.